### PR TITLE
12.0 mig hr expense cancel

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: hr_expense_cancel/models/account_payment.py [hunk 1]
```diff
@@ -8,6 +8,4 @@ class AccountPayment(models.Model):
     _inherit = "account.payment"
 
     expense_sheet_id = fields.Many2one(
-        comodel_name="hr.expense.sheet",
-        string="Expense sheet",
-    )
+        "hr.expense.sheet", string="Expense sheet")
```
Comment (pedrobaeza) [inline hr_expense_cancel/models/account_payment.py]: Why do you change this without need?

File: hr_expense_cancel/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -1,3 +1 @@
-This module allows you to cancel and correct expenses. It adds a cancel button
-on the expense sheet that undo reconciliations and delete payments and journal
-entries.
+This module allows you to cancel and correct expenses. It adds a cancel button on the expense sheet that undo reconciliations and delete payments and journal entries.
```
Comment (pedrobaeza) [inline hr_expense_cancel/readme/DESCRIPTION.rst]: Why changing this without need? And the change is incorrect at RST syntax level

File: hr_expense_cancel/readme/INSTALL.rst [hunk 3]
```diff
@@ -1,5 +1,2 @@
 It's assumed that for each expense sheet, the *payments* that are reconciled
-with their *journal entry* were created and reconciled by the *Expense Tracker*
-module automatically. Therefore, when this module is installed, these payments
-will automatically be associated with said expense sheet so that they can be
-deleted in case the expense sheet is canceled.
+with their *journal entry* were created and reconciled by the *Expense Tracker* module automatically. Therefore, when this module is installed, these payments will automatically be associated with said expense sheet so that they can be deleted in case the expense sheet is canceled.
```
Comment (pedrobaeza) [inline hr_expense_cancel/readme/INSTALL.rst]: And this? The change is incorrect at RST syntax level

File: hr_expense_cancel/readme/USAGE.rst [hunk 4]
```diff
@@ -1,14 +1,10 @@
 To use this module, you need to:
 
-#. Go to *Expenses -> My Expenses -> Expenses to Submit* and create a new
-   *Expense* with *Employee (to reimburse)* checked on the field *Payment By*
+#. Go to *Expenses -> My Expenses -> Expenses to Submit* and create a new *Expense* with *Employee (to reimburse)* checked on the field *Payment By*
```
Comment (pedrobaeza) [inline hr_expense_cancel/readme/USAGE.rst]: Don't change this if not needed

File: hr_expense_cancel/views/hr_expense_views.xml [hunk 5]
```diff
@@ -2,7 +2,6 @@
 <!-- Copyright 2019 Tecnativa - Ernesto Tejeda
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
-
```
Comment (pedrobaeza) [inline hr_expense_cancel/views/hr_expense_views.xml]: Why removing these lines?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please don't do unneeded changes. In a migration, you shouldn't change the "style", but the needed code for making it work

File: hr_expense_cancel/models/account_payment.py (hunk 1 above)
Comment (ghost) [reply hr_expense_cancel/models/account_payment.py]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/DESCRIPTION.rst (hunk 2 above)
Comment (ghost) [reply hr_expense_cancel/readme/DESCRIPTION.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/INSTALL.rst (hunk 3 above)
Comment (ghost) [reply hr_expense_cancel/readme/INSTALL.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/USAGE.rst (hunk 4 above)
Comment (ghost) [reply hr_expense_cancel/readme/USAGE.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/views/hr_expense_views.xml (hunk 5 above)
Comment (ghost) [reply hr_expense_cancel/views/hr_expense_views.xml]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/models/account_payment.py (hunk 1 above)
Comment (danhvophuong) [reply hr_expense_cancel/models/account_payment.py]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/DESCRIPTION.rst (hunk 2 above)
Comment (danhvophuong) [reply hr_expense_cancel/readme/DESCRIPTION.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/INSTALL.rst (hunk 3 above)
Comment (danhvophuong) [reply hr_expense_cancel/readme/INSTALL.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/readme/USAGE.rst (hunk 4 above)
Comment (danhvophuong) [reply hr_expense_cancel/readme/USAGE.rst]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

File: hr_expense_cancel/views/hr_expense_views.xml (hunk 5 above)
Comment (danhvophuong) [reply hr_expense_cancel/views/hr_expense_views.xml]: Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

**Comment by pedrobaeza** [comment]
No need to reply each of my comments. Just drop a comment saying "Changes done". Please squash your changes in one commit.

**Comment by pedrobaeza** [comment]
And check Travis status

**Comment by ghost** [comment]
Squashed commits and checked Travis status

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request closed without merging**